akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingGetStatsSpec.scala

```diff
@@ -144,23 +144,19 @@ abstract class ClusterShardingGetStatsSpec extends MultiNodeSpec(ClusterSharding
 
     "trigger sharded actors" in {
       runOn(controller) {
-        val region = ClusterSharding(system).shardRegion(shardTypeName)
-
         within(10.seconds) {
           awaitAssert {
             val pingProbe = TestProbe()
             // trigger starting of 2 entities on first and second node
             // but leave third node without entities
-            (1 to 6).filterNot(_ % 3 == 0).foreach(n ⇒ region.tell(Ping(n), pingProbe.ref))
+            List(1, 2, 4, 6).foreach(n ⇒ region.tell(Ping(n), pingProbe.ref))
             pingProbe.receiveWhile(messages = 4) {
               case Pong ⇒ ()
-            }
+            }.size === 4
```

Comment:
what does this do (if it was `should ===`) that is not detected by `receiveWhile(messages = 4)`?